volunteers/utils.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
+
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+
+
+def send_welcome_email(email, name):
+    sg = SendGridAPIClient(settings.SENDGRID_API_KEY)
+
+    message = Mail(
+        from_email=settings.DEFAULT_FROM_EMAIL,
+        to_emails=email,
+    )
```

Comment:
recebi o e-mail lindinho, porém, precisamos configurar o "sender name", que é o nome que aparece lá na caixa de inbox e hoje está só  "voluntaria".

ideal seria "Equipe do Mapa do Acolhimento".

vi nesse stack overflow como faz: https://stackoverflow.com/questions/67968102/sender-name-in-sendgrid-python-package

Reply:
@vivianedias boa! Revisões feitas